plugin/evm/vm_test.go

```diff
@@ -3204,3 +3206,83 @@ func TestSignatureRequestsToVM(t *testing.T) {
 	err = vm.Network.AppRequest(context.Background(), ids.GenerateTestNodeID(), 1, time.Now().Add(60*time.Second), requestBytes)
 	require.NoError(t, err)
 }
+
+// Test that the PruneWarpDB config flag applies as expected
+func TestPruneWarpDB(t *testing.T) {
+	// Because the chain is being reinitialized, metrics need to be disabled and upgrades skipped to allow initialization
+	t.Skip("no skipable upgrades")
```

Comment:
Should we remove `Skip` ? It seems that this test fails for me if I remove it atm

Reply:
Probably better to remove it, also since the unit test is being moved to the backend this entire unit test can probably be removed anyways